### PR TITLE
chore: Add npm test step CI

### PR DESCRIPTION
**Pull request by mzottolaqovery**

### CODE REVIEW

File: src/schemas/ClusterRequest.yaml [hunk 1]
```diff
@@ -30,7 +30,6 @@ properties:
     description: the instance type to be used for this cluster. The list of values can be retrieved via the endpoint /{CloudProvider}/instanceType
   kubernetes:
     $ref: ./enums/Kubernetes.yaml
-    default: "MANAGED"
```
Comment (mzottolaqovery) [inline src/schemas/ClusterRequest.yaml:33]: put the default to `Kubernetes.yaml`

File: src/schemas/Healthcheck.yaml [hunk 2]
```diff
@@ -3,7 +3,5 @@ nullable: false
 properties:
   readiness_probe:
     $ref: './Probe.yaml'
-    nullable: true
```
Comment (mzottolaqovery) [inline src/schemas/Healthcheck.yaml:6]: :warning: We need to check if it breaks something

File: src/schemas/Healthcheck.yaml [hunk 3]
```diff
@@ -3,7 +3,5 @@ nullable: false
...
   liveness_probe:
     $ref: './Probe.yaml'
-    nullable: true
```
Comment (mzottolaqovery) [inline src/schemas/Healthcheck.yaml:9]: :warning: We need to check if it breaks something

File: src/schemas/InviteMemberRequest.yaml [hunk 4]
```diff
@@ -6,7 +6,6 @@ properties:
     type: string
   role:
     $ref: './enums/InviteMemberRole.yaml'
-    description: deprecated
```
Comment (mzottolaqovery) [inline src/schemas/InviteMemberRequest.yaml:9]: Description already in the `InviteMemberRole.yaml`

File: src/schemas/InviteMemberResponse.yaml [hunk 5]
```diff
@@ -13,7 +13,6 @@
          format: email
        role:
          $ref: './enums/InviteMemberRole.yaml'
-         description: deprecated
```
Comment (mzottolaqovery) [inline src/schemas/InviteMemberResponse.yaml:16]: Description already in the InviteMemberRole.yaml

File: src/schemas/OrganizationEventResponse.yaml [hunk 6]
```diff
@@ -15,16 +15,12 @@ properties:
   target_name:
     type: string
   target_type:
-    type: string
```
Comment (mzottolaqovery) [inline src/schemas/OrganizationEventResponse.yaml:18]: type already in the `OrganizationEventTargetType.yaml`

File: src/schemas/OrganizationEventResponse.yaml [hunk 7]
```diff
@@ -15,16 +15,12 @@ properties:
...
     $ref: './enums/OrganizationEventTargetType.yaml'
   sub_target_type:
-    type: string
-    nullable: true
```
Comment (mzottolaqovery) [inline src/schemas/OrganizationEventResponse.yaml:22]: ⚠️ We need to check if it breaks something

File: src/schemas/OrganizationEventResponse.yaml [hunk 8]
```diff
@@ -15,16 +15,12 @@ properties:
...
     $ref: './enums/OrganizationEventSubTargetType.yaml'
   change:
     type: string
   origin:
-    type: string
```
Comment (mzottolaqovery) [inline src/schemas/OrganizationEventResponse.yaml:27]: type already in the OrganizationEventOrigin.yaml

File: src/schemas/SecretResponse.yaml [hunk 9]
```diff
@@ -19,13 +19,10 @@ allOf:
       service_id:
         type: string
         format: uuid
-        description: present only for `BUILT_IN` variable
```
Comment (mzottolaqovery) [inline src/schemas/SecretResponse.yaml:22]: This description is no longer true

File: src/schemas/SecretResponse.yaml [hunk 10]
```diff
@@ -19,13 +19,10 @@ allOf:
...
       service_name:
         type: string
-        description: present only for `BUILT_IN` variable
```
Comment (mzottolaqovery) [inline src/schemas/SecretResponse.yaml:25]: This description is no longer true

File: src/schemas/enums/LinkedServiceType.yaml [hunk 11]
```diff
@@ -1,7 +1,5 @@
 type: string
-description: |
-  type of the service (application, database, job, gateway...)  
-  present only for `BUILT_IN` variable
```
Comment (mzottolaqovery) [inline src/schemas/enums/LinkedServiceType.yaml:4]: This description is no longer true

**Comment by mzottolaqovery** [comment]
Hey @Qovery/frontends , I fixed the errors on open api highlighed by `npm run test`  (see first commit https://github.com/Qovery/qovery-openapi-spec/pull/465#discussion_r1347630158).
This should not break generated clients but just in case, you can take a look at those comments:
* https://github.com/Qovery/qovery-openapi-spec/pull/465#discussion_r1347620300
* https://github.com/Qovery/qovery-openapi-spec/pull/465#discussion_r1347620370
* https://github.com/Qovery/qovery-openapi-spec/pull/465#discussion_r1347622453
* https://github.com/Qovery/qovery-openapi-spec/pull/465#discussion_r1347630158

They are all related to a `nullable` property that was defined at the same level of defining `$ref`. The `nullable` property is now defined in the target enums, so it should be ok.

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**